model/src/main/scala/ch.epfl.scala.index.model/Release.scala

```diff
@@ -130,6 +130,22 @@ case class Release(
     s"compile group: '$groupId', name: '$artifactId', version: '$version'"
   }
 
+  /**
+   * string representation for mill dependency
+   * @return
+   */
+  def millInstall = {
+    def addResolver(r: Resolver) =
+      r.url map (url => s"""MavenRepository("${url}")""")
+    val artifactOperator = if (isNonStandardLib) ":" else "::"
+    List(
+      Some(
+        s"""ivy"${maven.groupId}$artifactOperator${reference.artifact}:${reference.version}""""
+      ),
+      resolver flatMap addResolver
```

Comment:
I’m not sure we should generate the “Mill” tab at all, if the `addResolver` method returns `None`.

Reply:
Why? In the [issue](**https://github.com/scalacenter/scaladex/issues/543**) @Baccata requested only the dependency. I can add an if guard in that case, but I think that it is more convenient if you can just copy paste the dependency even without a resolver.